src/robusta/core/sinks/robusta/dal/supabase_dal.py

```diff
@@ -142,6 +143,16 @@ def persist_scan(self, enrichment: Enrichment):
                 raise Exception(msg)
 
     def persist_finding(self, finding: Finding):
+        mute_finding = any(
```

Comment:
Why do we mute the finding?
We want the timeline to still behave the same, don't we?